doc/guide/scripting.rst

```diff
@@ -6,15 +6,33 @@
 Scripting Interface
 *******************
 
-Need some basic details here of how to load models and data via script, evaluate
-them at given parameter values and run bumps fits.
+Preparing your environment
+==========================
 
-The key functions are :func:`.core.load_model` for loading the
+You can use python scripts to load and plot your data, create SAS models and fit parameters. You can save a script to a file such as `example/model.py` and run
+it later. However, this requires a scripting environment with the correct packages installed. Either use the SasView application itself (versions after 5.0.5), as
+both bumps and sasmodels are included as part of the distribution, so for
+example::
+
+    $ sasview model.py
```

Comment:
`… To run the script you can use the SasView application (versions after 5.0.5), as both … distribution. So for example::`

Caveat: I don't know if we are putting the sasview binary on the system path in windows (or if we should…). If it is not there then we need to include the full path to sasview in the example, and that's going to be ugly.

Also, I should have used a windows cmd-style prompt rather than a unix prompt. For consistency we should also change `$ pip/python/bumps` below.

Reply:
Unfortunately, we stopped creating an environment variable for SasView after 3.1.0. So I've changed the text to read:
```
(Note that it may be necessary to first add the folder path to sasmodels/sasview
to your *Path* environment variable for this to work; set PATH=%PATH%;C:\your\path\\here\)
```
On reflection, setting PYTHONPATH didn't seem the appropriate place?

I have also changed a number of the prompts. Please check to see if you think the right prompt is now showing for the right environment!